[2024-11-27T04:18:20.135+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-27T04:18:20.160+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: amazon_recommendation_pipeline_elt.load_data manual__2024-11-27T04:18:13.071416+00:00 [queued]>
[2024-11-27T04:18:20.166+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: amazon_recommendation_pipeline_elt.load_data manual__2024-11-27T04:18:13.071416+00:00 [queued]>
[2024-11-27T04:18:20.167+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 4
[2024-11-27T04:18:20.176+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_data> on 2024-11-27 04:18:13.071416+00:00
[2024-11-27T04:18:20.183+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'amazon_recommendation_pipeline_elt', 'load_data', 'manual__2024-11-27T04:18:13.071416+00:00', '--job-id', '98', '--raw', '--subdir', 'DAGS_FOLDER/amazon_recommendation_dag.py', '--cfg-path', '/tmp/tmpz491xkr6']
[2024-11-27T04:18:20.184+0000] {standard_task_runner.py:91} INFO - Job 98: Subtask load_data
[2024-11-27T04:18:20.186+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=11301) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-27T04:18:20.187+0000] {standard_task_runner.py:63} INFO - Started process 11303 to run task
[2024-11-27T04:18:20.216+0000] {task_command.py:426} INFO - Running <TaskInstance: amazon_recommendation_pipeline_elt.load_data manual__2024-11-27T04:18:13.071416+00:00 [running]> on host bece637e0b85
[2024-11-27T04:18:20.273+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='amazon_recommendation_pipeline_elt' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-27T04:18:13.071416+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-27T04:18:13.071416+00:00'
[2024-11-27T04:18:20.274+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-27T04:18:20.275+0000] {milvus_load_task.py:24} INFO - Connecting to Milvus...
[2024-11-27T04:18:30.287+0000] {milvus_load_task.py:28} ERROR - Error connecting to Milvus: <MilvusException: (code=2, message=Fail connecting to server on localhost:19530, illegal connection params or server unavailable)>
[2024-11-27T04:18:30.287+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-27T04:18:30.288+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-27T04:18:30.294+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=amazon_recommendation_pipeline_elt, task_id=load_data, run_id=manual__2024-11-27T04:18:13.071416+00:00, execution_date=20241127T041813, start_date=20241127T041820, end_date=20241127T041830
[2024-11-27T04:18:30.332+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-27T04:18:30.343+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-27T04:18:30.345+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
